src/parseTools.js

```diff
@@ -744,6 +744,18 @@ function expectToReceiveOnModule(name) {
   return INCOMING_MODULE_JS_API.has(name);
 }
 
+// Return true if the user requested that a library symbol be included
+// wither via DEFAULT_LIBRARY_FUNCS_TO_INCLUDE or EXPORTED_RUNTIME_METHODS.
```

Comment:
Does this handle the case where it is needed because it is a dependency? (If not, why is it enough in the usage in this PR? I'm missing something I think.)

Reply:
No, since `isSymbolNeeded` is called then the library files themselves we read in, we don't have a way to support that today (Since dependencies are not processed until the libraries have all been read).

Reply:
i.e. we don't currently have a way to modify JS library functions according to what other functions exist in the dependency graph.

Reply:
Sorry, I'm not sure what you mean about modifying JS library functions. I think maybe my question wasn't clear. I'll clarify it at the location where I am confused, maybe that will help.